### PR TITLE
Refactor after simple flow removal

### PR DESCRIPTION
**Pull request by rcerven**
[STONEBLD-2612](https://issues.redhat.com//browse/STONEBLD-2612)

### Checklist:
 - PR has reference to the issue(s) it resolves
 - Write / update unit tests
 - Write / update integration (envtest) tests
 - Ensure there is an issue for e2e tests if needed
 - Ensure `make test` passes
 - Ensure test coverage hasn't decreased
 - Test code changes manually
 - Update readme and documentation
 - Write PR description that highlights overall changes and add usage examples if applicable

### CODE REVIEW

**Comment by tkdchen** [comment]
Can you add more info in the commit message about what is done in this refactor?

File: controllers/suite_util_test.go [hunk 1]
```diff
@@ -35,7 +35,8 @@ import (
 	appstudiov1alpha1 "github.com/konflux-ci/application-api/api/v1alpha1"
 
 	. "github.com/konflux-ci/build-service/pkg/common"
-	"gopkg.in/yaml.v2"
+	"sigs.k8s.io/yaml"
+	//"gopkg.in/yaml.v2"
```
Comment (mmorhun) [inline controllers/suite_util_test.go:39]: Leftover

File: controllers/component_build_controller_pipeline.go [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	appstudiov1alpha1 "github.com/konflux-ci/application-api/api/v1alpha1"
+	tektonapi "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	tektonapi_v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	oci "github.com/tektoncd/pipeline/pkg/remote/oci"
+
+	// vs
+	"sigs.k8s.io/yaml"
+	//"gopkg.in/yaml.v2"
```
Comment (mmorhun) [inline controllers/component_build_controller_pipeline.go:35]: Leftover

File: controllers/component_build_controller_pipeline.go [hunk 3]
```diff
@@ -0,0 +1,536 @@
...
+	"path/filepath"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	appstudiov1alpha1 "github.com/konflux-ci/application-api/api/v1alpha1"
+	tektonapi "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	tektonapi_v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	oci "github.com/tektoncd/pipeline/pkg/remote/oci"
+
+	// vs
```
Comment (mmorhun) [inline controllers/component_build_controller_pipeline.go:33]: Leftover

File: controllers/component_build_controller_pipeline.go [hunk 4]
```diff
@@ -0,0 +1,536 @@
+/*
+Copyright 2024 Red Hat, Inc.
```
Comment (mmorhun) [inline controllers/component_build_controller_pipeline.go:2]: I think it all started earlier

**Review by mmorhun — APPROVED** [review]

File: controllers/suite_util_test.go (hunk 1 above)
Comment (rcerven) [reply controllers/suite_util_test.go:39]: removed

File: controllers/component_build_controller_pipeline.go (hunk 2 above)
Comment (rcerven) [reply controllers/component_build_controller_pipeline.go:35]: removed

File: controllers/component_build_controller_pipeline.go (hunk 4 above)
Comment (rcerven) [reply controllers/component_build_controller_pipeline.go:2]: changed

File: controllers/component_build_controller_pac_repository.go [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+
+		} else {
+			log.Error(err, "failed to get Component PaC repository object", l.Action, l.ActionView)
+			return err
+		}
+	}
+	return nil
+}
+
+func (r *ComponentBuildReconciler) getNamespace(ctx context.Context, name string) (*corev1.Namespace, error) {
```
[the remark this answers is not in the document]
Comment (rcerven) [reply controllers/component_build_controller_pac_repository.go:144]: well it is used only in ensurePaCRepository, so I've moved it here :)

**Comment by rcerven** [comment]
> Can you add more info in the commit message about what is done in this refactor?

added

**Pull request merged**